order-book.js

```diff
@@ -0,0 +1,99 @@
+const reconcileOrder = (existingBook, incomingOrder) => {
+
+  switch (existingBook.length) {
+    case 0:
+      return toBook(existingBook, incomingOrder)
+
+    default:
+      return determineOrder(existingBook, incomingOrder)
+  }
```

Comment:
This should be an `if` you would only use a switch for multiple cases not a single case like this